graviti/openapi/data.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the data."""
+from typing import Any, Dict, Optional
+from urllib.parse import urljoin
+
+from graviti.openapi.requests import open_api_do
+
+
+def list_draft_data(
```

Comment:
`list_draft_data` and `list_commit_data` is almost the same, can we decrease the duplicate code？

Reply:
Here, we can not modify the arguments. The duplicate code is the judgment on whether the argument is None or "". It doesn't feel necessary to merge them. Almost every API has this code. Or we need to put this judgement to open-api-do?